samples/apps/copilot-chat-app/webapi/Skills/OpenApiSkills/JiraSkill/Model/CommentAuthor.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace SemanticKernel.Service.Skills.OpenApiSkills.JiraSkill.Model;
+
+public class CommentAuthor
+{
+    /// <summary>
+    /// Gets or sets the Comment Author's display name.
+    /// </summary>
+    [JsonPropertyName("displayName")]
+    public string displayName { get; set; }
```

Comment:
Style consistency: please use PascalCase for all public properties (i.e., "DisplayName"). Same comment for all model classes.